tembo-operator/src/cloudnativepg/hibernate.rs

```diff
@@ -142,18 +145,26 @@ pub async fn reconcile_cluster_hibernation(cdb: &CoreDB, ctx: &Arc<Context>) ->
         }
     }
 
-    // Build the hibernation patch we want to apply to disable the CNPG cluster.
+    // Stop CNPG reconciles for hibernated instances
```

Comment:
let's add a note here about why it is important that we dont set turn off reconciliation until hibernation is complete.